pkg/datapath/linux/probes/probes.go

```diff
@@ -386,10 +388,137 @@ func HaveBoundedLoops() error {
 }
 
 // HaveFibIfindex checks if kernel has d1c362e1dd68 ("bpf: Always return target
-// ifindex in bpf_fib_lookup") which is 5.10+. This got merged in the same kernel
-// as the new redirect helpers.
-func HaveFibIfindex() error {
-	return features.HaveProgramHelper(ebpf.SchedCLS, asm.FnRedirectPeer)
+// ifindex in bpf_fib_lookup") which is 5.10+ (and backports). It does so by
+// checking that ifindex has been updated as a result of the RIB lookup even
+// though the bpf_fib_lookup failed (no L2 resolution for the NH)
+func HaveFibIfindex() (err error) {
+	defer func() {
+		if err != nil && !errors.Is(err, ebpf.ErrNotSupported) {
+			log.WithError(err).Fatal("failed to probe HaveFibIfindex")
+		}
+
+		if errors.Is(err, ebpf.ErrNotSupported) {
+			log.Info("HaveFibIfindex NOT supported")
```

Comment:
Nit: make this debug?

Reply:
I don't have a strong opinion.

Moving it to `Dbg` together with fixing the errors of `govet` (thx, wasn't aware of that `make` target)

Reply:
Done